manual/root_macros_and_shared_libraries/index.md

```diff
@@ -199,385 +180,35 @@ The `$ROOTSYS/include` directory is automatically appended to the include path.
 To get the include path, type:
 
 ```
-   root[] .include
+root [0] .include
 ```
 
 To append the include path, type:
 
 ```
-   root[] .include $HOME/mypackage/include
+root [0] .include $HOME/mypackage/include
 ```
 
-Append the following line in the ROOT macro to include the include path:
+Add the following line in the ROOT macro to append a new path to the existing include paths:
 
 ```
-   gSystem->AddIncludePath(" -I$HOME/mypackage/include")
+gSystem->AddIncludePath(" -I$HOME/mypackage/include")
 ```
 
 To overwrite an existing include path, type:
 
 ```
-   gSystem->SetIncludePath(" -I$HOME/mypackage/include")
+gSystem->SetIncludePath(" -I$HOME/mypackage/include")
 ```
 
 To add a static library that should be used during linking, type:
 
 ```
-   gSystem->AddLinkedLibs("-L/my/path -l*anylib*");
+gSystem->AddLinkedLibs("-L/my/path -l*anylib*");
 ```
 
 For adding a shared library, you can load it before you compile the ROOT macros, by
```

Comment:
You can also add a shared library by loading it before compiling the ROOT macros, by doing:

Reply:
Right, sounds better